src/Models/User/User.php

```diff
@@ -263,4 +264,16 @@ public function shiftsUpdated(): HasMany
     {
         return $this->hasMany(Shift::class, 'updated_by');
     }
+
+    public function getDisplayNameAttribute(): string
+    {
+        if (
+            config('display_full_user_name')
+            && !empty(trim($this->personalData->first_name . $this->personalData->last_name))
+        ) {
+            return trim($this->personalData->first_name . ' ' . $this->personalData->last_name);
```

Comment:
The names should be trimmed separately here to avoid duplicated white spaces.

```suggestion
            return trim($this->personalData->first_name) . ' ' . trim($this->personalData->last_name);
```